### ISSUE TITLE
TypeError when creating submission in control panel

### ISSUE THREAD

**Issue by michaelhue** [verbb/formie#609]
**Description**
I wanted to test the new "Create submissions from control panel" feature. Trying to save a submission results in the following error:

```
TypeError
verbb\formie\elements\Submission::getFieldContext(): Return value must be of type string, null returned
```

**Steps to reproduce**
1. Click on "New submission" in control panel
2. Try to save a new submission

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.21.1
- Craft version: Craft Pro 3.7.17
- Multi-site: No

**Additional context**

Full stack trace:
```
TypeError: verbb\formie\elements\Submission::getFieldContext(): Return value must be of type string, null returned in /vendor/verbb/formie/src/elements/Submission.php:467
Stack trace:
#0 /vendor/craftcms/cms/src/base/Element.php(4293): verbb\formie\elements\Submission->getFieldContext()
#1 /vendor/craftcms/cms/src/base/Element.php(1732): craft\base\Element->fieldByHandle('form')
#2 /vendor/verbb/formie/src/controllers/SubmissionsController.php(205): craft\base\Element->__get('form')
#3 [internal function]: verbb\formie\controllers\SubmissionsController->actionSaveSubmission()
#4 /vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#5 /vendor/yiisoft/yii2/base/Controller.php(181): yii\base\InlineAction->runWithParams(Array)
#6 /vendor/yiisoft/yii2/base/Module.php(534): yii\base\Controller->runAction('save-submission', Array)
#7 /vendor/craftcms/cms/src/web/Application.php(277): yii\base\Module->runAction('formie/submissi...', Array)
#8 /vendor/craftcms/cms/src/web/Application.php(586): craft\web\Application->runAction('formie/submissi...', Array)
#9 /vendor/craftcms/cms/src/web/Application.php(256): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#10 /vendor/yiisoft/yii2/base/Application.php(392): craft\web\Application->handleRequest(Object(craft\web\Request))
#11 /web/index.php(26): yii\base\Application->run()
#12 /Users/michael/.composer/vendor/laravel/valet/server.php(219): require('/Users/michael/...')
#13 {main}
```

**Comment by engram-design** [comment]
Sorry about that! Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.21.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.22](https://github.com/verbb/formie/releases/tag/1.4.22)
